print-sctp.c

```diff
@@ -366,6 +472,55 @@ struct sctpCWR{
   nd_uint32_t TSN_reduced_at;
 };
 
+/* RE-CONFIG Parameters */
+static const struct tok RE_CONFIG_parameters[] = {
+	{ OUT_SSN_RESET,	"OUT SSN RESET"	},
+	{ IN_SSN_RESET,		"IN SSN RESET"	},
+	{ SSN_TSN_RESET,	"SSN/TSN Reset"	},
+	{ RE_CONFIG_RES,	"RESP"		},
+	{ ADD_OUT_STREAM_REQ,	"ADD OUT STREAM"},
+	{ ADD_IN_STREAM_REQ,	"ADD IN STREAM" },
+	{ 0, NULL }
+};
+
+static const struct tok results[] = {
+	{ 0,	"Success - Nothing to do"		},
+	{ 1,	"Success - Performed"			},
+	{ 2,	"Denied"				},
+	{ 3,	"Error - Wrong SSN"			},
+	{ 4,	"Error - Request already in progress"	},
+	{ 5,	"Error - Bad Sequence Number"		},
+	{ 6,	"In progress"				},
+};
```

Comment:
`{ 0, NULL }` missing.

Reply:
There is a key of value zero already, 
```C
static const struct tok results[] = {
       { 0,    "Success - Nothing to do"               },
```
I guess that I don't need add `{0, NULL}` for this structure?

Reply:
See the tok2strbuf() function.